.travis.yml

```diff
@@ -13,4 +18,6 @@ before_script:
         else
             PYTEST_ADDOPTS="-m 'not optional'"
         fi
-script: make PYTEST_ADDOPTS="${PYTEST_ADDOPTS}" test-docker
+script:
+    - docker version
```

Comment:
I think this is no longer needed, but maybe you can keep it and move it to `install`, so it doesn't show highlighted in the test output.